tcl.html.markdown

```diff
@@ -30,9 +32,9 @@ Various features of Tcl stand out:
 * A threading model recognized as robust and easy to use
 
 
-If Lisp is a list processor, then Tcl is a string processor.  All values are
-strings.  A list is a string format.  A procedure definition is a string
-format.  To achieve performance, Tcl internally caches structured
+Tcl is a Lisp built on the foundation of string processing.  All values are
```

Comment:
Upon googling this is a borderline defensible statement, but if you leave "Tcl is a Lisp" in there it's just going to be reverted by some uptight lisper.